tests/nightly/test_large_array.py

```diff
@@ -455,6 +455,20 @@ def npy_instance_norm(data, gamma, beta, axis, eps=1E-5):
         assert_almost_equal(out, out_nd.asnumpy(), forward_check_eps,
                             forward_check_eps)
 
+    def check_multi_lars():
+        lrs = nd.random_normal(shape=(2**32 + 1, 1))
+        weights_sum_sq = nd.random_normal(shape=(2**32 + 1, 1))
+        grads_sum_sq = nd.random_normal(shape=(2**32 + 1, 1))
+        wds = nd.random_normal(shape=(2**32 + 1, 1))
+        eta = .1
+        eps = .9
+
+        out = nd.multi_lars(lrs=lrs, weights_sum_sq=weights_sum_sq, grads_sum_sq=grads_sum_sq,
+                            wds=wds, eta=eta, eps=eps)
+
+        assert out.shape[0] == 4294967297
```

Comment:
Can you make this a constant and replace in the code ?

Reply:
Done!